test/fuzz/CMakeLists.txt

```diff
@@ -0,0 +1,35 @@
+# Copyright (C) 2023 Intel Corporation
+# Part of the Unified-Runtime Project, under the Apache License v2.0 with LLVM Exceptions.
+# See LICENSE.TXT
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+function(add_fuzz_test name)
+    set(TEST_TARGET_NAME fuzztest-${name})
+    add_ur_executable(${TEST_TARGET_NAME}
+        ${ARGN})
+    target_link_libraries(${TEST_TARGET_NAME}
+        PRIVATE
+        ${PROJECT_NAME}::loader
+        ${PROJECT_NAME}::headers
+        ${PROJECT_NAME}::common
+        -fsanitize=fuzzer -fprofile-instr-generate -fcoverage-mapping)
+    add_test(NAME ${TEST_TARGET_NAME}
+        COMMAND ${TEST_TARGET_NAME} -max_total_time=600 -seed=1 -shrink=1 -verbosity=1
```

Comment:
We should have a set of predefined input data files we know pass for CI, and a separate setup where we run the fuzzer for longer periods of time. Even with a fixed seed, running this on every build might make the CI flaky.

Reply:
OK, and in the predefined input the fuzzer should provide just functions parameters and not generate API calls on the fly, right?

Reply:
yes, it should be a file that contains all the bytes you use in your program to create some sequence of operations.

Reply:
Done